### ISSUE TITLE
Add Support for More .env Files

### ISSUE THREAD

**Issue by laugharn** [prisma/prisma#3226]
## Problem

I primarily use Prisma with Next.js, and since Next 9.4 they've implemented automatic .env detection, based on Create React App's pattern: https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used

When doing local development, my DATABASE_URL variable is kept in .env.local, which isn't found by default when I run `prisma generate` so Prisma commands tend to require some overhead/repetition in my repo.

## Suggested solution

Support for the additional .env files used by the popular JS meta frameworks.

## Alternatives
A set of local development-specific Prisma commands using dotenv-cli

**Comment by rolivegab** [comment]
I would like to see this feature too.
It'll be very useful for develop with Next.js, for example.
See next.js .env configuration here: https://nextjs.org/docs/basic-features/environment-variables

**Comment by flybayer** [comment]
Yes! We also need this for Blitz. Here's another issue for the same thing: https://github.com/prisma/prisma/issues/3720

**Comment by elkevinwolf** [comment]
This would be very helpful, currently, my solution is proxying `prisma` through an npm script that uses `dotenv-cli`. Happy to help with a PR if needed.

**Comment by mikeedwards83** [comment]
Bump on this request

**Comment by saevarb** [comment]
This would really be a nice thing to fix because this causes some pretty annoying(and very avoidable) friction when using next.js.

Next.js uses a convention like .env.local for local development variables while prisma uses .env.development. This means that to ensure consistency, you might want to involve something like `dotenv-flow`.. but then it suddenly becomes unclear how this plays with next.js special treatment of environment variables(which they do for security reasons).

**Comment by drewhoffer** [comment]
Any update on this? Would really like to use other files depending on my environment.

**Comment by liran319** [comment]
Any update about this?
I just add DATABASE_URL in to nextjs .env file, but got `error: Environment variable not found: DATABASE_URL.` when run `npx prisma migrate dev `

**Comment by hassan-debug** [comment]
i have my .env.local file but still prisma is unable to read environment variables
datasource db {
  provider = "postgresql"
  url = env("POSTGRES_PRISMA_URL") // uses connection pooling

}

**Comment by janpio** [comment]
Yes, Prisma does not support this out of the box yet. We have documentation on how you can manually achieve that: https://www.prisma.io/docs/guides/development-environment/environment-variables/using-multiple-env-files
